Scripts/Common/execWPS.sh

```diff
@@ -24,13 +24,29 @@ METDATA=${METDATA:-''} # folder to store metgrid data on disk (has to be absolut
 # N.B.: leave undefined to skip disk storage; defining $METDATA will set "ldisk = True" in pyWPS
 # real.exe
 RUNREAL=${RUNREAL:-1} # whether to run real.exe
-REALIN=${REALIN:-"${METDATA}"} # location of metgrid files
 REALTMP=${REALTMP:-"./metgrid"} # in case path to metgrid data is too long
 RAMIN=${RAMIN:-1} # copy input data to ramdisk or read from HD
 REALOUT=${REALOUT:-"${WORKDIR}"} # output folder for WRF input data
 RAMOUT=${RAMOUT:-1} # write output data to ramdisk or directly to HD
 REALLOG="real" # log folder for real.exe
 REALTGZ="${RUNNAME}_${REALLOG}.tgz" # archive for log folder
+if [[ "${RAMIN}" == 0 ]] && [[ -z  "${METDATA}" ]] 
+then
+  REALIN=${REALIN:-"${PYDATA}"}
```

Comment:
in case RAMIN==0 and -z METDATA, set RAMDATA=$WORKDIR/ram_disk/ and RAMIN=1
But only do this if we run pyWPS! so move this after RUNPYWPS==1

Reply:
I fixed this according to our talks.